scripts/build_packages.sh

```diff
@@ -106,30 +99,16 @@ for var in "${VARIATION_ARRAY[@]}"; do
         echo Building package for channel ${CHANNEL} to ${PLATFORM_ROOT}
 
         pushd ${PLATFORM_ROOT}
-        tar --exclude=tools -zcf ${PKG_ROOT}/${GATE_PREFIX}node_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.tar.gz * >/dev/null 2>&1
+        tar --exclude=tools -zcf ${PKG_ROOT}/node_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.tar.gz * >/dev/null 2>&1
         cd bin
-        tar -zcf ${PKG_ROOT}/${GATE_PREFIX}install_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.tar.gz updater update.sh >/dev/null 2>&1
+        tar -zcf ${PKG_ROOT}/install_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.tar.gz updater update.sh >/dev/null 2>&1
         if [ $? -ne 0 ]; then
             echo "Error creating tar file for package ${PLATFORM}.  Aborting..."
             exit 1
         fi
 
         cd ${PLATFORM_ROOT}/tools
-        tar -zcf ${PKG_ROOT}/${GATE_PREFIX}tools_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.tar.gz * >/dev/null 2>&1
+        tar -zcf ${PKG_ROOT}/tools_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.tar.gz * >/dev/null 2>&1
         popd
-
```

Comment:
build_release.sh calls build_packages.sh and expects that to build the .deb
we could turn it off here with a variable test, e.g. `if [ ! -z "${BUILD_DEB}" ];` or so